content/rs/references/rladmin.md

```diff
@@ -1,30 +1,697 @@
+        [ max_simultaneous_backups <size> ] 
+        [ watchdog_profile <cloud | local-network> ] 
+        [ slave_ha <enabled/disabled> ] 
+        [ slave_ha_grace_period <seconds> ] 
+        [ slave_ha_cooldown_period <seconds> ] 
+        [ slave_ha_bdb_cooldown_period <seconds> ] 
+        [ continue_on_error ]
+```
+
+| Optional Parameters | Description |
+| - | - |
+| repl_diskless | Enables/disables diskless replication (can be overwritten per database) |
+| default_redis_version | Default Redis version for new databases |
+| redis_provision_node_threshold | Memory (in MBs) needed to provision a new database |
+| redis_migrate_node_threshold | Memory (in MBs) needed to migrate a database between nodes |
+| redis_provision_node_threshold_percent | Memory (in percentage) needed to provision a new database |
+| redis_migrate_node_threshold_percent | Memory (in percentage) needed to migrate a database between nodes |
+| max_simultaneous_backups | Number of backups allowed to happen at once |
+| watchdog_profile | Enables pre-configured watchdog profiles ( cloud or local-network) |
+| slave_ha | Enables/disables slave high availability |
+| slave_ha_grace_period | Time (in seconds) between when a node fails and when slave high availability starts relocating shards to another node |
+| slave_ha_cooldown_period | Time (in seconds) after shard relocation during which the slave high availibity mechanism can't relocate to another node  |
+| slave_ha_bdb_cooldown_period | Time (in seconds) after shard relocation during which databases can't be relocated to another node |
+| continue_on_error | Skips shards that can't be reached |
+
+Redis cluster watchdog supports two pre-configured profiles:
+-  `cloud` profile is suitable for common cloud environments. It has a higher tolerance for network jitter.
+- `local-network`profile is suitable for dedicated LANs and has a better failure detection and failover times.
+
+### `upgrade`
+
+`rladmin upgrade` schedules a restart of the master and slave processes to upgrade the Redis software for a specific database instance.
+
+```text
+rladmin upgrade db <db:id | name> 
+        [ preserve_roles ] 
+        [ keep_current_version ] 
+        [ discard_data ] 
+        [ force_discard ]
+```
+
+| Optional Parameters | Description |
+| - | - |
+| preserve_roles | Performs an additional failover to guarantee roles of masters are preserved |
+| keep_current_version | Upgrades to a new patch release, not to the latest major.minor version |
+| discard_data | Indicates that data will not be saved after the upgrade |
+| force_discard | Forces `discard_data` if there is replication or persistence enabled |
+
+### `verify`
+
+`rladmin verify balance` prints a cluster shard balance report.
+
+```text
+rladmin verify balance [ node <id> ]
+```
+
+`rladmin verify rack_aware` prints a cluster rack aware verification report.
+
+```text
+rladmin verify rack_aware
+```
+
+## Use the rladmin shell
```

Comment:
I would lead with this info.

Reply:
I don't think the rladmin shell adds much value over just running individual rladmin commands besides the fact that it saves you a few keystrokes. Are there any capabilities the rladmin shell gives users that I missed in the list below?

Reply:
I think that it is much easier and user-friendly to run from the shell, especially for beginner users that may not even know that it is an option.

Reply:
I will make sure to mention the shell when writing "how-to" articles. When I can expand each section into its own page, we will be able to add more detail.